test/core/test_select.c

```diff
@@ -0,0 +1,24 @@
+#include <sys/select.h>
```

Comment:
Should we call this `test_pipe_select` maybe?

Or should we add this to the existing `test/unistd/pipe.c`?